Code/Tools/pluglib/always.h

```diff
@@ -85,6 +85,15 @@ void operator delete(void *p, size_t size) noexcept;
 #pragma warning ( disable: 1 ) // last line of file ends without a newline
 #endif
 
+// Provide a portable definition for __forceinline when not compiling with MSVC.
+#if !defined(_MSC_VER) && !defined(__forceinline)
+	#if defined(__GNUC__) || defined(__clang__)
+		#define __forceinline inline __attribute__((always_inline))
```

Comment:
Since we have C++20, we can do [`[[msvc::forceinline]]`](https://learn.microsoft.com/en-us/cpp/cpp/attributes?view=msvc-170#msvcforceinline) as an attribute.